test/run.js

```diff
@@ -0,0 +1,82 @@
+var driver = require("./driver.js");
+require("../inject")(require("acorn"));
+require("./tests-jsx.js");
+
+function group(name) {
+  if (typeof console === "object" && console.group) {
+    console.group(name);
+  }
+}
+
+function groupEnd() {
+  if (typeof console === "object" && console.groupEnd) {
+    console.groupEnd(name);
+  }
+}
+
+function log(title, message) {
+  if (typeof console === "object") console.log(title, message);
+}
+
+var stats, modes = {
+  Normal: {
+    config: {
+      parse: require("acorn").parse
+    }
+  }/*,
```

Comment:
Given that we don't support loose mode for JSX, this commented part can be safely removed.